cdap-e2e-tests/src/e2e-test/features/Hub/DesignTime.feature

```diff
@@ -28,3 +28,40 @@ Feature: Hub - Design time scenarios
     Then Click on Hub Menu
     Then Enter the text in search tab "invalidMessage"
     Then Verify that user is getting an error message: "invalid.message_hub"
+
+  @TS-HUB-DESIGN-03
+  Scenario: Verify that the user is able to find the plugin with appropriate name in search tab
+    When Open Datafusion Project to configure pipeline
+    Then Click on Hub Menu
+    Then Enter the text in search tab "Anaplan"
+    Then Verify that "Anaplan plugins" plugin is displayed on the Hub page
+
+  @TS-HUB-DESIGN-04
+  Scenario: Verify that the user is getting an error message when deploying plugin multiple times
```

Comment:
Change scenario: Verify the error message when user is deploying the same plugin multiple times.

Reply:
done